### ISSUE TITLE
Create QUBES Ambassador page

### ISSUE THREAD

**Issue by mdlama** [qubeshub/hubzero-cms#119]
[Text for Ambassadors page](https://docs.google.com/document/d/1PZQHnCqvqmonsPG36przYLj_XbuM1rWecV3U-STtQhc/edit)

This page would not necessarily be strictly FMN focused, but it will be right now.

@trisketz:  Hope it's okay to assign you to this!  Do you think you could throw something together before launch on March 4?

**Comment by mdlama** [comment]
Hmmm, okay, just created the page on dev.  Can you screenshot the 500 error and upload here please?  Curious to know what is going on...

**Comment by trisketz** [comment]
This is the message I always get when I try and create a page on dev.
![screen shot 2018-02-23 at 10 57 08 am](https://user-images.githubusercontent.com/21963909/36606142-67dbbbe4-1888-11e8-8dfa-fe4a771a328c.png)

**Comment by trisketz** [comment]
I should note, the only fields I'm filling out are title, alias, and I write a line of text. Everything else I leave blank/as is.

**Comment by trisketz** [comment]
The page is all set, let me know if I'm missing anything. I believe we just need to link it somewhere.

**Comment by mdlama** [comment]
Looks great.  Can we move the text up a bit in the main banner image?  Feels like gravity is too strong there.  :)

<img width="1057" alt="screen shot 2018-02-25 at 11 58 07 am" src="https://user-images.githubusercontent.com/4877315/36644090-3be8d55c-1a23-11e8-9be0-f2e74e4b11f3.png">

Created a PDF of the page to share with the FMN group.  Thanks, Jenny!

**Comment by trisketz** [comment]
Oops, sorry about that! Forgot I had put that styling in there when I was working on fallbacks. It's all fixed to be in the center. I also made the changes from the feedback. Thanks!

**Comment by trisketz** [comment]
The ambassador page is up on QA, did we decide where this link is going to reside?

**Comment by mdlama** [comment]
How about we put a blurb about it and link to it after the example FMNs on the community page? (so at the bottom of this image)

<img width="1421" alt="screen shot 2018-03-02 at 9 45 50 am" src="https://user-images.githubusercontent.com/4877315/36904412-a0d2f004-1dfe-11e8-9a44-571cdb18ab8e.png">

**Comment by trisketz** [comment]
Not a bad idea. I will go ahead and do that. I feel like if we were to do that, that would be a good place to mention something about the mentor bridge program as well. But I don't think we have a page for that?

**Comment by mdlama** [comment]
Good point on the mentor bridge program.  That can be added later, I think.  The mentor bridge program actually has a group that we could point to.  This would be a good thing to bring up on Monday at the full meeting.

**Comment by trisketz** [comment]
Ok, I've got a one liner blurb up on the community page. I'm wondering if we need to link it to a menu item or not. It just looks so lonely ;)
<img width="1415" alt="screen shot 2018-03-02 at 3 29 55 pm" src="https://user-images.githubusercontent.com/21963909/36922807-d9257b4a-1e2e-11e8-97b1-f880c52ddd1d.png">

**Comment by mdlama** [comment]
Awesome - can you have have the "Learn More" link go here instead?  

/community/fmns/ambassadors

I've set it up so that the sub-menu will show up when you go to this page, and the breadcrumbs will show that QUBES Ambassadors is under the FMN page.  I'm not sure we want a sub-menu item for it right now.  Good thinking to link it into the menu, somehow, though!
